controllers/sts.go

```diff
@@ -216,15 +228,22 @@ func createDSStatefulSet(ds *directoryv1alpha1.DirectoryService, sts *apps.State
 								},
 								{
 									Name:      "secrets",
-									MountPath: "/opt/opendj/secrets",
+									MountPath: "/opt/opendj/pem-keys-directory/ssl-key-pair",
+									SubPath:   "ssl-key-pair-combined.pem",
+								},
+								{
+									Name:      "secrets",
+									MountPath: "/opt/opendj/pem-keys-directory/master-key",
```

Comment:
Initially, I had `master-key-pair`. However, DS was looking for `master-key` alias so I had to rename it to that. 
I didn't want to add code in setup.sh just to change this with the fear it would just introduce bugs. I just decided to use the `master-key` default name instead

Reply:
ok no problem. Just checking that you didn't just miss it.